src/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Why is there a `.gitignore` in the source folder? I don't see why we need this

Reply:
weird, not sure how it got added and didn't even realize it! i'm pretty sure i didn't add it 😂